client/main.lua

```diff
@@ -35,12 +35,50 @@ local NpcData = {
     CountDown = 180
 }
 
+local CabParkingZone = nil
+local CabBossZone = nil
+
+local taxiPed = nil
+
 -- events
+
+AddEventHandler('onResourceStop', function(resourceName)
+    if resourceName == GetCurrentResourceName() and QBCore.Functions.GetPlayerData() ~= {} then
```

Comment:
`~= {}` doesn't work, instead use `table.type(QBCore.Functions.GetPlayerData()) ~= 'empty'`

Reply:
And you can invert both if statements here and combine them into `if resource ~= GetCurrentResourceName() or table.type(QBCore.Functions.GetPlayerData()) == 'empty' or not Config.UseTarget then return end`